### ISSUE TITLE
Stratum client - object constructor has no method 'on'

### ISSUE THREAD

**Issue by epsicoin** [pocesar/node-stratum#10]
Hello,

I used this in my code

var stratum = require('stratum');
var Client = stratum.Client;
var client = Client.create();
client.on('mining.error', function(msg, socket) {
      console.log(msg);
});

and when i launch i throw this error.

Any idea ?

**Comment by pocesar** [comment]
```js
var stratum = require('stratum');
var client = stratum.Client.$create();
client.on('mining.error', function(msg, socket) {
   console.log(msg);
});
```

**Comment by epsicoin** [comment]
Thanks for your reply, but it's strange, i still got the same issue

**Comment by epsicoin** [comment]
TypeError: Object #<constaructor> has no method 'on' at Object.<anonymous>

**Comment by pocesar** [comment]
ah my bad, EventEmitter3 1.x had a breaking change, I'll have to update the lib. for now, change the package.json manuall to use EventEmitter3 0.x

**Comment by epsicoin** [comment]
Works better Thanks !

**Comment by pocesar** [comment]
I've updated the code and released as 0.2.4
